w3-ipni.md

```diff
@@ -0,0 +1,237 @@
+# W3 IPNI Protocol
+
+![status:wip](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+## Authors
+
+- [olizilla], [Protocol Labs]
+
+# Abstract
+
+For IPNI we assert that we can provide batches of multihashes by signing "Advertisements". 
+
+With an inclusion claim, a user asserts that a CAR contains a given set of multihashes via a car index.
+
+This spec describes how to merge these two concepts by adding an `ipni/offer` capability to submit an inclusion claim as an IPNI Advertisement.
```

Comment:
`ipni/offer` implies an `ipni/accept` fx per our own conventions...

It might be good to have a `ipni/accept` task that is invoked when the advert has been written. The receipt might include the advert (C)ID and an identifier for the chain that it is included in.

Reply:
Who is the consumer of that receipt? This would be a copy of the `Announce` message, that has the advertisement CID and the peerID and addresses of where the chain is available from. The peerID (publisher ID) would identify which chain the ad is on.